h5p-bildetema/src/components/SearchPage/useSearchResults.tsx

```diff
@@ -14,8 +14,9 @@ import { useReducer } from "react";
 import { useNewDBContext } from "common/hooks/useNewDBContext";
 import { LanguageCode } from "common/types/LanguageCode";
 import { useBackendUrlContext } from "common/hooks/useBackendUrlContext";
-import { getImageUrl } from "common/utils/image/image.utils";
-import { getAudioFiles } from "common/utils/audio/audio.utils";
+// import { getImageUrl } from "common/utils/image/image.utils";
+// import { getAudioFiles } from "common/utils/audio/audio.utils";
```

Comment:
Should we remove these imports if we don't need them?

Reply:
Yes!